tests/versions/apm_server.yml

```diff
@@ -1,4 +1,4 @@
 APM_SERVER:
   - master
-  - master --oss
+  - master --apm-server-oss
```

Comment:
not sure if APM Server will have only one binary too

Reply:
IIUC, beats didn't transition to a different license but Elasticsearch and Kibana, so I'd assume no changes in that particular regard